python/spec/fixtures/requirements/pip_compile_uv_header.txt

```diff
@@ -0,0 +1,22 @@
+# This file was autogenerated by uv via the following command:
+#    uv pip compile --output-file pip_compile_uv_header.txt unpinned_uv.in -P attrs==18.1.0
```

Comment:
```suggestion
#    uv pip compile --output-file pip_compile_uv_header.txt unpinned_uv.in attrs==18.1.0
```

Reply:
Actually it should remove the value too.


```suggestion
#    uv pip compile --output-file pip_compile_uv_header.txt unpinned_uv.in
```

But looks like this is not being used for tests at all.

Reply:
@skshetry maybe there's a bug in uv 0.2.25 cause the header does include the value when the input is

```
uv pip compile --output-file pip_compile_uv_header.txt unpinned_uv.in -P attrs==18.1.0
```

Reply:
@edgarrmondragon, can you share `uv --version` please? I just tried this locally and it seems to work fine.

Reply:
```console
uv --version
uv 0.2.25 (d1010228b 2024-07-15)
```

Reply:
Sorry, now I see. Did not notice that easily. I'll investigate.

Reply:
Fixed in https://github.com/astral-sh/uv/pull/5111.

Reply:
@skshetry's suggestion is now the correct one :)